### ISSUE TITLE
一个view  只下了2个UIlabel   用的masonry布局  骨架显示了4个 请问为什么呢   

### ISSUE THREAD

**Issue by littleZhangqq** [tigerAndBull/TABAnimated#86]
### New Issue Checklist

* [ ] I have read the [Documentation](https://github.com/tigerAndBull/TABAnimated/tree/master/Documents)
* [ ] I have searched for a similar issue in the [project](https://github.com/rs/TABAnimated/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. iOS / macOS
 Platform Version        | e.g. 12.0 / 10.14.0
 TABAnimated Version      | e.g. 5.0.0 / 4.4.0
 Xcode Version           | e.g. Xcode 11 / Xcode 10
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by tigerAndBull** [comment]
请提供更多信息，或许是分割线、滚动条、narrow等造成的，框架不会主动为你生成view的。
